api.md

```diff
@@ -58,6 +58,52 @@ your use case. Users will *rarely* want or need to use this command.
 converted to a hypertable (via `create_hypertable`), but must similarly
 be run only on an empty hypertable.
 
+**Space partitions**: The use of additional partitioning is a very
```

Comment:
There is a `warning` above this section in the add_dimension description that references best practices.  Maybe you want to remove that reference as well?

Reply:
Ok, yes - removing the warning. Seems repetitive